src/OptiHPLCHandler/empower_api_core.py

```diff
@@ -115,47 +133,85 @@ def logout(self) -> None:
         self.session_id = None
         logger.debug("Logout successful")
 
-    def get(self, endpoint: str) -> requests.Response:
+    def _requests_wrapper(
+        self, method: str, endpoint: str, body: Optional[dict], timeout
+    ) -> requests.Response:
         """
-        Get data from Empower.
+        Wrapper for requests.
 
-        :param endpoint: The endpoint to get data from.
+        :param method: The method to use.
+        :param endpoint: The endpoint to use.
+        :param body: The body to use.
         """
+
+        def _request_with_timeout(method, endpoint, header, body, timeout):
+            try:
+                response = requests.request(
```

Comment:
Just return the result straight away.

Reply:
Hmm, that feels dirty, somehow. But it also seems like a better way of doing it.